extension/models/Errors/TokenRenewError.js

```diff
@@ -0,0 +1,15 @@
+const ETOKENRENEW = 'ETOKENRENEW'
+
+class TokenRenewError extends Error {
+  constructor (displayMessage) {
+    super('Error renewing customer access token.')
+    this.code = ETOKENRENEW
+
+    this.displayMessage = null
```

Comment:
no longer a thing anymore as far as I know, was planned to be implemented in PWA, but never was or will be